pkg/api/api.proto

```diff
@@ -232,6 +232,7 @@ enum ParameterType {
     INT = 2; /// Int type. Use "Max/Min".
     DISCRETE = 3; /// Discrete number type. Use "List" as float.
     CATEGORICAL = 4; /// Categorical type. Use "List" as string.
+    RANGE = 5; /// Range type. Use "Max/Min" and "Step" parameter.
```

Comment:
Do we really need Range ParameterType?
Only add step field, We can merge it to INT and Double.
If the range is not set, the default range is 1 for INT, 0 for Double.

Reply:
The incentive for adding Range ParameterType is that, to the best of our knowledge, none of the NAS algorithms supports continuous search space. All the search space must be discretized. 
Yes, what you said makes good sense. We will delete Range and add step in int and double.